.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
The .idea configuration folder shall not be in the repo, please make sure git was initialized correctly